primitives/core/hashing/Cargo.toml

```diff
@@ -25,7 +25,7 @@ sp-std = { version = "5.0.0", default-features = false, path = "../../std" }
 default = ["std"]
 std = [
 	"digest/std",
-	"blake2/std",
+	"blake2b_simd/std",
```

Comment:
Are there any tests to ensure that this is 100% compatible with the other crate?  
I remember for the signatures there were edge cases that were handled differently across crates.

Reply:
We have some tests, although not any which do it exhaustively.

I think it's unlikely that this will differ (this is just boring old hashing), and `blake2b_simd` is made by the same person who came up with BLAKE2 in the first place.

But I can probably run a quick fuzz test to see if they're equivalent.

Reply:
Okay, I quickly whipped out a fuzz test and, 30 million iterations later, as expected it found nothing.